clang/lib/Format/UnwrappedLineParser.h

```diff
@@ -171,6 +171,8 @@ class UnwrappedLineParser {
   void parseRequiresClause(FormatToken *RequiresToken);
   void parseRequiresExpression(FormatToken *RequiresToken);
   void parseConstraintExpression();
+  void parseCXXExportBlock();
```

Comment:
```suggestion
  void parseCppExportBlock();
```
This would be more consistent.

Reply:
Ah sure; I’m used to `CXX` being used everywhere in Clang haha.